cmake/ContourThirdParties.cmake

```
@@ -124,11 +145,20 @@ else()
     set(THIRDPARTY_BUILTIN_termbench "(bench-headless disabled)")
 endif()
 
+set(BOXED_CPP_VERSION "1.2.2")
```

```suggestion
set(BOXED_CPP_MINIMAL_VERSION "1.2.2")
```
maybe we can rename it so we see what we mean here